olive/passes/onnx/inc_quantization.py

```diff
@@ -208,9 +264,148 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
         config.update(get_external_data_config())
         return config
 
+    def _set_eval_func(self, accuracy_metric, sub_type):
+        # set eval_func for INC according to Olive accuracy metric
+        def eval_func(model):
+            # eval_func for Intel® Neural Compressor quantization take model as input,
```

Comment:
I think this can be simplified by creating a temporary `ONNXModel` and calling the evaluator model. This way, we won't need need to create a custom inference session or custom metric that takes in an onnx model proto. 

Please refer to this commit for a sample implementation https://github.com/microsoft/Olive/commit/3f2b6fe3081691b9dee97a8b12f773df68d310b6 

Please feel free to cherry pick the commit if it meets your requirements.

Reply:
thanks, we will follow up your suggestion on that.